examples/node18-prisma-rest-express/README.md

```diff
@@ -0,0 +1,15 @@
+# Node Prisma Example
+
+To run this example, first [install the `kraft` CLI tool](https://unikraft.org/docs/cli).
+
+Then, clone this repository and `cd` into this directory.
+To deploy this application on KraftCloud, invoke:
+
+```
+kraft cloud deploy -M 512 -p 443:3000 .
```

Comment:
Is there a way to test this locally or is it intended only for `kraft cloud` use?